packages/trackerless-network/protos/NetworkRpc.proto

```diff
@@ -164,6 +164,12 @@ message StreamPartitionInfo {
   string id = 1;
   repeated dht.PeerDescriptor controlLayerNeighbors = 2;
   repeated dht.PeerDescriptor contentDeliveryLayerNeighbors = 3;
+  repeated ContentDeliveryRtt rtts = 4;
```

Comment:
Yeah implemented it like this because of compatibility. Although, as the InfoResponse should contain the version it should not be an issue to have a different schema